src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,74 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private static final int INDEX_OF_NAME = 1;
+    private static final int INDEX_OF_WORKING_HOURS = 2;
+    private static final int INDEX_OF_INCOME_PER_HOUR = 3;
+    private static final String LINE_SEPARATOR = System.lineSeparator();
+    private static final DateTimeFormatter
+            DATE_TIME_FORMAT = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+    private static final String DELIMITER_SPACE = " ";
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        int[] salary = countSalary(names, data, dateFrom, dateTo);
+        StringBuilder salaryInfo = new StringBuilder("Report for period "
+                + dateFrom
+                + " - "
+                + dateTo
+                + LINE_SEPARATOR);
```

Comment:
lets use StringBuilder properly
```suggestion
        StringBuilder salaryInfo = new StringBuilder("Report for period ")
                .append(dateFrom)
                .append(" - ")
                .append(dateTo)
                .append(LINE_SEPARATOR));
```